docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,272 @@
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+#### Verify the Transaction
+
+You can now pass this txID `r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1` into [examples/avm/getTx.ts](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/getTx.ts), plus other similar network settings, then you can run
+
+```zsh
+$ ts-node examples/avm/getTx.ts
+```
+
+which returns:
+
+```js
+{
+  unsignedTx: {
+    networkID: 1,
+    blockchainID: '2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM',
+    outputs: [],
+    inputs: [ [Object] ],
+    memo: '0x',
+    destinationChain: '11111111111111111111111111111111LpoYY',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction:
+
+- Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To export AVAX from the X-Chain to the C-Chain, select [`evm/buildExportTx-xchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts)
+
+- Make necessary changes as above for private key and network settings.
+
+- You can change the amount of AVAX to send by editing the _BN_ variable: `avaxAmount`. The sample code assigns this as `1e7` or `10000000` (0.01 AVAX)
+
+The fee here will only be for exporting the asset. The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs stay after getting exported but before being imported.
+
+```ts
+let avaxAmount: BN = new BN(1e7);
+let fee: BN = baseFee.div(new BN(1e9));
+fee = fee.add(new BN(1e6));
+```
+
+:::tip
+When exporting AVAX, be sure to send enough to support import fees (constant .001 AVAX). Sending less than `1e6` or `1000000` (0.001 AVAX) may cause the import txn to fail.
+:::
+
+The following sequence demonstrates how to send AVAX back to the X-Chain:
+
+```zsh
+avalanchejs $ ts-node examples/evm/buildExportTx-xchain-avax.ts
+Success! TXID: UAez3DTv26qmhKKFDvmQTayaXTPAVahHenDKe6xnUMhJbKuxc
+avalanchejs $ ts-node examples/avm/buildImportTx-cchain.ts
```

Comment:
Please separate this file out in its own paragraph and describe in details what needs to be changed before running it.